sql/planbuilder/scalar.go

```diff
@@ -131,8 +131,9 @@ func (b *Builder) buildScalar(inScope *scope, e ast.Expr) (ex sql.Expression) {
 		return c.scalarGf()
 	case *ast.FuncExpr:
 		name := v.Name.Lowered()
-
-		if isAggregateFunc(name) && v.Over == nil {
+		if name == "name_const" {
```

Comment:
similar to the other PR, what are the pros/cons of registering with the function handlers, vs manually doing it here, vs a new expression. I like the current approach better than new expression, but is there a reason we shouldn't add to list of func handlers?

Reply:
I didn't put this in the function registry cause it would never be used. 
Additionally, I had to make sure avoid having the `FunctionName()` or `String()` methods accidentally be used as the alias.